webapp/app/routes.py

```diff
@@ -353,6 +352,13 @@ def ambassadorMaterial():
             props=AmbassadorInfoMaterialProps(plausible_domain=Config.PLAUSIBLE_DOMAIN).camelized_dict(), 
             component='AmbassadorInfoMaterialPage')
 
+    @app.route('/krankheitskosten', methods=['GET'])
+    @add_caching_headers
+    def krankheits_kosten_info():
```

Comment:
Also here method name in English? We do the same with the ambassador page, the actual URL is in German but the rest in English